spec/features/users/sign_in_spec.rb

```diff
@@ -60,7 +60,7 @@
   scenario 'user can see and use password visibility toggle', js: true do
     visit new_user_session_path
 
-    find('#pw-toggle-0', visible: false).trigger('click')
+    find('.checkbox').click
```

Comment:
`.trigger('click')` is a Poltergeist/PhantomJS hack that is no longer needed with Chrome webdriver.